WalletWasabi/WabiSabi/Backend/Rounds/ConcurrentDictionaryValueCollectionView.cs

```diff
@@ -0,0 +1,29 @@
+using System;
```

Comment:
You can delete this file, use simple collections with lock.

Reply:
this is just to avoid modifying all of the tests that violate abstraction boundaries, it should be removed but requires a lot of cleanups in the tests and WabiSabiFactory